### PR TITLE
enable cinder/ceph with redhat

### PR DESCRIPTION
**Pull request by chengtcli**

### CODE REVIEW

File: roles/ceph-common/defaults/main.yml [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
     - hdparm
-  ceph_pkgs:
-    - ceph
-    - ceph-common
-  osd_pkgs:
+  osd_dep_pkgs:
     - parted
   bcache_pkgs:
-    - bcache-tools
+      - bcache-tools
```
Comment (paulczar) [inline roles/ceph-common/defaults/main.yml]: is there too many spaces here ?

File: roles/ceph-common/meta/main.yml [hunk 2]
```diff
@@ -6,7 +6,9 @@ dependencies:
     when: logging.enabled|default(True)|bool
     service: ceph
     logdata: "{{ ceph.logs }}"
-  - role: apt-repos
+  - role: repos
     repos:
       - repo: 'deb {{ apt_repos.ceph.repo }} {{ ansible_lsb.codename }} main'
         key_url: '{{ apt_repos.ceph.key_url }}'
+    when: os == 'ubuntu'
+# as redhat already has ceph repo added, we don't need to add it here
```
Comment (paulczar) [inline roles/ceph-common/meta/main.yml]: we should probably do something for centos at some point,  but for now this is fine.

**Comment by paulczar** [comment]
overall this is looking pretty good.

**Comment by nirajdp76** [comment]
LGTM

File: roles/ceph-common/defaults/main.yml (hunk 1 above)
Comment (chengtcli) [inline roles/ceph-common/defaults/main.yml]: right. done

**Comment by chengtcli** [comment]
I have do a rebase on `centos` branch, and have tested on tardis-sl with following case:
1. run deploy by using env of envs/example/ci-ceph-redhat/
2. deployment is successful.(VM create with flavor ci_ceph_swift has /dev/vdb mounted on /mnt, need to umount it manually)
3. create VM and volume, attach volume to VM.
4. write volume by using `dd`.  10GB data is write into ceph cluster in less than 50 seconds.

Issues:
VM has no ip address, we may have some issues with neutron.

There are some more test cases I will have to cover, but let's merge this big PR first. I have some more PRs which are related to ceph.

**Pull request merged**